pkg/simple/client/multicluster/options.go

```diff
@@ -67,4 +71,7 @@ func (o *Options) AddFlags(fs *pflag.FlagSet, s *Options) {
 
 	fs.StringVar(&o.AgentImage, "agent-image", s.AgentImage, ""+
 		"This field is used when generating deployment yaml for agent.")
+
+	fs.IntVar(&o.ClusterControllerResyncSecond, "cluster-controller-resync-second", s.ClusterControllerResyncSecond,
```

Comment:
why not using fs.DurationVar ?

Reply:
I didn't realize that there is a `fs.DurationVar` function. It has been replaced.